README.md

```diff
@@ -1,4 +1,4 @@
-# th2 common library (Java) (3.23.0)
+# th2 common library (Java) (3.23.1)
```

Comment:
```suggestion
# th2 common library (Java) (3.24.0)
```
Also, add information about a new feature to `Release notes` part at the end of README